docs/docs/pyproject.md

```diff
@@ -173,6 +173,26 @@ include = ["CHANGELOG.md"]
 exclude = ["my_package/excluded.py"]
 ```
 
+## dev-only
+
+If set to `true`, this option entirely disables installing runtime dependencies in the development environment.
+This is useful when you do not care about whether it's possible to install the package under development in the development environment, and you don't want to mandate command-line flags to avoid attempting to do so.
+Use cases for this include running tests with `tox` (in which case the packages are unnecessary) or minimizing `pip` installs when building the project from a fresh checkout.
```

Comment:
Those use cases aren't crystal clear to me. I stiill don't understand, why don't you leave runtime dependencies empty, if you don't want to specify them within poetry? Or in other words, is there a clear answer to the following question:

- Why would you pretend specifying prod dependencies only to say poetry should completely ignore them?

Reply:
Because I'm creating an installable package that has runtime dependencies. So, I want to specify dependencies that are installed at runtime, and have them show up in the metadata of the packages produced by `poetry build`.

Reply:
Got ya, thanks!
Then it probably is a `install` / `add` / `remove` flag as it should be ignored for `build`. Unless, `poetry` would introduce a `tool.poetry.<good word for saying anything but build>` subsection, don't you think?